rllib/agents/trainer.py

```diff
@@ -558,11 +562,15 @@
     # Specify how to evaluate the current policy. This only has an effect when
     # reading offline experiences ("input" is not "sampler").
     # Available options:
-    #  - "wis": the weighted step-wise importance sampling estimator.
-    #  - "is": the step-wise importance sampling estimator.
-    #  - "simulation": run the environment in the background, but use
+    #  - "simulation": Run the environment in the background, but use
     #    this data for evaluation only and not for learning.
-    "input_evaluation": ["is", "wis"],
+    #  - Any subclass of OffPolicyEstimator, e.g.
+    #    ray.rllib.offline.estimators.is::ImportanceSampling or your own custom
+    #    subclass.
+    "input_evaluation": [
+        ImportanceSampling,
+        WeightedImportanceSampling,
```

Comment:
also just curious, why do we run both by default.

Reply:
Oh is that what this means? Same question then... why not run 1, giving users the option which one that they want to run?

Reply:
Short answer: It's always been this way :D 

Happy to change this default behavior, but let's do this in a different PR and only if we have a good reason to break this (maybe some users rely on both being there and don't even know, why these are always there).